integration-tests/gatsby-source-wordpress/__tests__/__snapshots__/index.js.snap

```diff
@@ -222,7 +237,7 @@ Object {
           "author": Object {
             "node": Object {
               "avatar": Object {
-                "url": "https://secure.gravatar.com/avatar/5fc9a2065017733c9d5f49a5ecd71d64?s=96&d=mm&r=g",
+                "url": "http://2.gravatar.com/avatar/e64c7d89f26bd1972efa854d13d7dd61?s=96&d=mm&r=g",
```

Comment:
What's the `2.gravatar` about?

Reply:
I believe this is either from the version of WP upgrading or because I switched from my personal account to a generic admin account in the seed db.